Code/Framework/AzToolsFramework/Tests/ComponentModeSwitcherTests.cpp

```diff
@@ -95,6 +95,42 @@ namespace UnitTest
         EXPECT_EQ(1, componentModeSwitcher->GetComponentCount());
     }
 
+        TEST_F(ComponentModeSwitcherTestFixture, InstantaneousChangeOfEntitySelectionUpdatesSwitcherCorrectly)
```

Comment:
nit: Indentation looks a little off here (there could be a few too many spaces?)